plenum/server/batch_handlers/three_pc_batch.py

```diff
@@ -5,15 +5,8 @@
 
 
 class ThreePcBatch:
-    def __init__(self,
-                 ledger_id,
-                 inst_id, view_no, pp_seq_no,
-                 pp_time,
-                 state_root, txn_root,
-                 primaries,
-                 valid_digests,
-                 has_audit_txn=True,
-                 original_view_no=None) -> None:
+    def __init__(self, ledger_id, inst_id, view_no, pp_seq_no, pp_time, state_root, txn_root, primaries, valid_digests,
+                 digest, has_audit_txn=True, original_view_no=None) -> None:
```

Comment:
I would rename it to `pp_digest` to disctinguish request's digests and PrePrepare's digest

Reply:
Sure thing, makes sense.